/usr/local/lib/python3.10/dist-packages/pytorch_lightning/loggers/wandb.py:391: There is a wandb run already in progress and newly created instances of `WandbLogger` will reuse this run. If this is not desired, call `wandb.finish()` before instantiating `WandbLogger`.
[32m[I 2024-05-14 02:34:56,964][39m A new study created in memory with name: no-name-83a19097-70ac-4baf-be14-c2ee38b73a3c
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
You are using a CUDA device ('NVIDIA GeForce RTX 4090') that has Tensor Cores. To properly utilize them, you should set `torch.set_float32_matmul_precision('medium' | 'high')` which will trade-off precision for performance. For more details, read https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html#torch.set_float32_matmul_precision
/usr/local/lib/python3.10/dist-packages/pytorch_lightning/callbacks/model_checkpoint.py:653: Checkpoint directory /content/murata_labo_exp/checkpoint exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
┏━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┳━━━━━━━━┓
┃[1m    [22m┃[1m Name                        [22m┃[1m Type             [22m┃[1m Params [22m┃
┡━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━╇━━━━━━━━┩
│ 0  │ bert                        │ BertModel        │ 89.1 M │
│ 1  │ classifiers                 │ ModuleList       │ 12.5 M │
│ 2  │ hidden_layer1               │ ModuleList       │ 13.6 M │
│ 3  │ hidden_layer2               │ ModuleList       │ 13.4 K │
│ 4  │ sigmoid                     │ Sigmoid          │      0 │
│ 5  │ criterion                   │ BCELoss          │      0 │
│ 6  │ metrics                     │ MetricCollection │      0 │
│ 7  │ metrics_per_label_accuracy  │ MetricCollection │      0 │
│ 8  │ metrics_per_label_precision │ MetricCollection │      0 │
│ 9  │ metrics_per_label_recall    │ MetricCollection │      0 │
│ 10 │ metrics_per_label_f1score   │ MetricCollection │      0 │
└────┴─────────────────────────────┴──────────────────┴────────┘
[1mTrainable params[22m: 33.2 M
[1mNon-trainable params[22m: 82.0 M
[1mTotal params[22m: 115 M
[1mTotal estimated model params size (MB)[22m: 460
[37mEpoch 0/7 [39m [90m━━━━━━━━━━━━━━━━━━━━━━━━━━━[39m [37m4/355[39m [37m0:00:00 • 0:01:04[39m [37m5.56it/s[39m [37mv_num: gm0l train/loss:     











































                                                                         [37m0.662                      




[37mValidation[39m [35m━━━━━━━━━━━━━━━━━━━━╸[90m━━━━━[39m [37m36/45  [39m [37m0:00:07 • 0:00:03[39m [37m4.48it/s





































                                                                         [37m0.663                      





[37mValidation[39m [35m━━━━━━━━━━━━━━━━━━━━━━━━[90m╺━[39m [37m42/45  [39m [37m0:00:09 • 0:00:01[39m [37m4.46it/s







































                                                                         [37m0.664                      





[37mValidation[39m [35m━━━━━━━━━━━━━━━━━━━━━━━╸[90m━━[39m [37m41/45  [39m [37m0:00:09 • 0:00:01[39m [37m4.46it/s





































                                                                          [37m0.661                     





[37mValidation[39m [35m━━━━━━━━━━━━━━━━━━━━━[90m╺━━━━[39m [37m37/45  [39m [37m0:00:08 • 0:00:02[39m [37m4.47it/s


































                                                                         [37m0.664                      





[37mValidation[39m [35m━━━━━━━━━━━━━━━━━━━━━━━━[90m╺━[39m [37m42/45  [39m [37m0:00:09 • 0:00:01[39m [37m4.45it/s




































                                                                          [37m0.666                     





[37mValidation[39m [35m━━━━━━━━━━━━━━━━━━━━━[90m╺━━━━[39m [37m37/45  [39m [37m0:00:08 • 0:00:02[39m [37m4.47it/s



































                                                                         [37m0.660                      





[37mValidation[39m [35m━━━━━━━━━━━━━━━━━━━━━━━╸[90m━━[39m [37m41/45  [39m [37m0:00:09 • 0:00:01[39m [37m4.46it/s






































                                                                         [37m0.656                      




[37mValidation[39m [35m━━━━━━━━━━━━━━━━━━━━╸[90m━━━━━[39m [37m36/45  [39m [37m0:00:07 • 0:00:03[39m [37m4.47it/s
Epoch 7, global step 2840: 'val_loss' reached 0.66012 (best 0.66012), saving model to '/content/murata_labo_exp/checkpoint/epoch=7.ckpt' as top 1
`Trainer.fit` stopped: `max_epochs=8` reached.
[33m[W 2024-05-14 02:49:38,370][39m Trial 0 failed with parameters: {'batch_size': 58, 'epoch': 4, 'hidden_size': 1015, 'hidden_size2': 835, 'chank_prev': 9} because of the following error: AttributeError("'MaltiLabelClassifierModel' object has no attribute 'validation_f1score'").
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/optuna/study/_optimize.py", line 196, in _run_trial
    value_or_values = func(trial)
  File "/content/murata_labo_exp/murata_labo_exp_src/exp9/main.py", line 591, in objective
    f1_score = model.validation_f1score
  File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py", line 1688, in __getattr__
    raise AttributeError(f"'{type(self).__name__}' object has no attribute '{name}'")
AttributeError: 'MaltiLabelClassifierModel' object has no attribute 'validation_f1score'
[33m[W 2024-05-14 02:49:38,371][39m Trial 0 failed with value None.
Error executing job with overrides: []
Traceback (most recent call last):
  File "/content/murata_labo_exp/murata_labo_exp_src/exp9/main.py", line 596, in main
    study.optimize(objective,n_trials=5)
  File "/usr/local/lib/python3.10/dist-packages/optuna/study/study.py", line 451, in optimize
    _optimize(
  File "/usr/local/lib/python3.10/dist-packages/optuna/study/_optimize.py", line 62, in _optimize
    _optimize_sequential(
  File "/usr/local/lib/python3.10/dist-packages/optuna/study/_optimize.py", line 159, in _optimize_sequential
    frozen_trial = _run_trial(study, func, catch)
  File "/usr/local/lib/python3.10/dist-packages/optuna/study/_optimize.py", line 247, in _run_trial
    raise func_err
  File "/usr/local/lib/python3.10/dist-packages/optuna/study/_optimize.py", line 196, in _run_trial
    value_or_values = func(trial)
  File "/content/murata_labo_exp/murata_labo_exp_src/exp9/main.py", line 591, in objective
    f1_score = model.validation_f1score
  File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py", line 1688, in __getattr__
    raise AttributeError(f"'{type(self).__name__}' object has no attribute '{name}'")
AttributeError: 'MaltiLabelClassifierModel' object has no attribute 'validation_f1score'
Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
[?25h